cime_config/testmods_dirs/allactive/pioroot1/shell_commands

```diff
@@ -0,0 +1 @@
+./xmlchange PIO_ROOT=1
```

Comment:
You should wrap this with an "if mach=sandiatoss3" or it will be set to 1 on all machines that run this test.

Reply:
ERP had a FAIL on anvil too, on Nov 20 with hash 15bf0dae8 on next (that had PIO_ROOT=1 just for sandiatoss3) -- https://my.cdash.org/viewTest.php?onlyfailed&buildid=1921288:
```
2020-11-20 03:36:36: memleak detected, memory went from 3470.250000 to 4019.950000 in 1 days
```
a 15.84% mem-usage increase. So the ERP testmod appears to be needed for all machines.

Reply:
oh ok.